soroban-env-host/src/test/stellar_asset_contract.rs

```diff
@@ -322,6 +323,30 @@ impl StellarAssetContractTest {
     }
 }
 
+fn check_event_name(test: &StellarAssetContractTest, e: HostEvent, contract: &Address, name: &str) {
+    assert!(e.event.type_ == ContractEventType::Contract && e.event.contract_id.is_some());
+
+    let address = ScAddress::Contract(e.event.contract_id.clone().unwrap());
```

Comment:
I would recommend using `assert_eq!(contract.to_sc_adress(), ScAddress::Contract(...)` - less code and better error message.

Reply:
Fixed